gui/wxpython/web_services/widgets.py

```diff
@@ -709,7 +710,9 @@ def CreateCmd(self):
 
         if 'srs' not in self.drv_props['ignored_params']:
             i_srs = self.params['srs'].GetSelection()
-            epsg_num = int(self.projs_list[i_srs].split(':')[-1])
+            srs = self.projs_list[i_srs].split(':')[-1]
+            epsg_num = int(''.join(re.findall('\d+', srs)))
```

Comment:
Hopefully last small request, could you please use raw string for the expression? Quoting from Python doc:

> This is complicated and hard to understand, so it’s highly recommended that you use raw strings for all but the simplest expressions.

Thanks!

Reply:
Yes of course, I fixed it.